Automattic/jetpack
# Jetpack Boost: Use TypeScript in Svelte

#### Changes proposed in this Pull Request:
So far we've been using TypeScript only partially. This PR enables TypeScript in all Svelte files and fixes any TypeScript issues that showed up afterward.

#### Other information:


#### Jetpack product discussion
We discussed this in the last sprint meeting.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1. Apply the patch
2. Build the plugin
3. Run e2e tests
4. Test the plugin to ensure nothing has changed
